packages/core/src/utils/fsUtil.js

```diff
@@ -2,7 +2,7 @@ const path = require('path');
 const fs = require('fs-extra');
 const ensurePosix = require('ensure-posix-path');
 
-const markdownFileExts = ['.md', '.mbd', '.mbdf'];
+const markdownFileExts = '.md';
```

Comment:
Will have to update [isMarkdownFileExt](https://github.com/MarkBind/markbind/blob/864900461b9bd1e46b069fecd8d5455a3da78ff7/packages/core/src/utils/fsUtil.js#L18) method as well since it is no longer an array. Can use `===` instead of `includes`.

Reply:
Includes will still work with strings, so there should be no issue with this particular case. Unless the change is for readability reasons?

![image](https://user-images.githubusercontent.com/19247856/154194460-10368275-80f3-4ff1-9e46-fbaeab9012bf.png)

Reply:
I believe this will give a false positive? Although unlikely, but files with extension `.m` will be returned as `true`.

Reply:
Ahh ok I see, will implement the change then!